connect-widget/app/src/shared/localStorage/custom-color.ts

```diff
@@ -1,10 +1,13 @@
 const colorLocalStorageKey = "custom-color";
 
-export const getCustomColor = (paramColor: string | null) => {
+export const getCustomColor = (paramColor: string | null, isDemo: boolean) => {
   const color = validateColor(paramColor);
 
   if (color) {
-    localStorage.setItem(colorLocalStorageKey, color);
+    if (!isDemo) {
```

Comment:
should this be `if (!isDemo && color)` ?

Reply:
I had it like that but then the color below wasnt getting returned

Reply:
nvm that's my bad
I totally missed the `if (color) {` in the original PR